gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.6-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.0-bin.zip
+networkTimeout=10000
```

Comment:
Why do we need this one explicitely? It looks like default settings already

Reply:
Don't know actually. It adds it automatically when i upgrade gradle via `.gradle wrapper --gradle-version 8.0.1`